### PR TITLE
docs: JK-9706: apply style updates to Kafka scripts quick start

### PR DESCRIPTION
**Pull request by jbyrne-redhat**
**Notes for reviewers**

- Applies [MAS style guidelines](https://docs.google.com/document/d/1TERH4XRH6La44hUQH88LUaax-35JH3dLXmceGJoLv7I/edit?pli=1#heading=h.v8qjgp8tzewa) to the Kafka scripts quick start
- Streamlines the text of the quick start where possible
- An HTML preview is available for review [here](http://file.emea.redhat.com/~jbyrne/JK-9706/README.html)

### CODE REVIEW

File: docs/kafka/kafka-bin-scripts-kafka/README.adoc [hunk 1]
```diff
@@ -129,25 +126,23 @@ endif::[]
...
 
 .Prerequisites
-ifndef::qs[]
-* You have the generated credentials for your service account. To reset the credentials, use the {service-accounts-url}[Service Accounts^] page in the *Application Services* section of the Red Hat Hybrid Cloud Console.
-* You've set the permissions for your service account to access the Kafka instance resources. To verify the current permissions, select your Kafka instance in the {product-kafka} web console and use the *Access* page to find your service account permission settings.
-endif::[]
+* You have the generated credentials for your service account. To reset the credentials, use the {service-accounts-url}[Service Accounts^] page.
+* You've set permissions for your service account to access resources in the Kafka instance. To verify the current permissions, select your Kafka instance in the {product-kafka} web console and click the *Access* tab. To learn more about setting permissions, see {base-url}{access-mgmt-url-kafka}[Managing account access in {product-long-kafka}^].
```
Comment (AedinC) [inline docs/kafka/kafka-bin-scripts-kafka/README.adoc:133]: Refer to the console with its long name on first usage in assembly or module.

File: docs/kafka/kafka-bin-scripts-kafka/README.adoc [hunk 2]
```diff
@@ -170,24 +165,23 @@ NOTE: {product-kafka} also supports the SASL/OAUTHBEARER mechanism for authentic
...
-You can use the `kafka-console-producer` script to produce messages to Kafka topics.
+In this task, you use the `kafka-console-producer` script to produce messages to a Kafka topic.
 
 .Prerequisites
 
 * You have a running Kafka instance in {product-kafka}.
 ifndef::qs[]
-* You have the bootstrap server endpoint for your Kafka instance. To relocate the server endpoint, select your Kafka instance in the {product-kafka} web console, select the options menu (three vertical dots), and click *Connection*.
-* You've set the permissions for your service account to access the Kafka instance resources. To verify the current permissions, select your Kafka instance in the {product-kafka} web console and use the *Access* page to find your service account permission settings.
+* You have the bootstrap server endpoint for your Kafka instance. To get the server endpoint, select your Kafka instance in the {product-kafka} web console, select the options icon (three vertical dots), and click *Connection*.
```
Comment (AedinC) [inline docs/kafka/kafka-bin-scripts-kafka/README.adoc:174]: See above comment.

File: docs/kafka/kafka-bin-scripts-kafka/README.adoc [hunk 3]
```diff
@@ -228,7 +216,7 @@ The preceding example uses the SASL/PLAIN authentication mechanism with the cred
 >Third message
 ----
 
-. Keep the producer running to use later when you create a consumer.
+. Keep the producer running to use later, when you create a consumer.
```
Comment (AedinC) [inline docs/kafka/kafka-bin-scripts-kafka/README.adoc:219]: This sentence structure is a little clunky. Suggest 'Keep the producer running so you can use it later when you create a consumer.'

File: docs/kafka/kafka-bin-scripts-kafka/README.adoc [hunk 4]
```diff
@@ -170,24 +165,23 @@ NOTE: {product-kafka} also supports the SASL/OAUTHBEARER mechanism for authentic
 == Producing messages using Kafka scripts
 
 [role="_abstract"]
-You can use the `kafka-console-producer` script to produce messages to Kafka topics.
+In this task, you use the `kafka-console-producer` script to produce messages to a Kafka topic.
 
 .Prerequisites
 
 * You have a running Kafka instance in {product-kafka}.
```
Comment (AedinC) [inline docs/kafka/kafka-bin-scripts-kafka/README.adoc:172]: Use the full product name upon first usage in every module.

**Review by AedinC — APPROVED** [review]
Great set up of updates John. Really makes things very clear for the reader. Just had a few very minor comments.

File: docs/kafka/kafka-bin-scripts-kafka/README.adoc (hunk 1 above)
Comment (jbyrne-redhat) [reply docs/kafka/kafka-bin-scripts-kafka/README.adoc:133]: Woops, yes - thank you!

File: docs/kafka/kafka-bin-scripts-kafka/README.adoc (hunk 3 above)
Comment (jbyrne-redhat) [reply docs/kafka/kafka-bin-scripts-kafka/README.adoc:219]: Yes, agree, that's clearer.

File: docs/kafka/kafka-bin-scripts-kafka/README.adoc (hunk 4 above)
Comment (jbyrne-redhat) [reply docs/kafka/kafka-bin-scripts-kafka/README.adoc:172]: Yes - thanks for catching.

**Pull request merged**